input/sources/tev-000065.yaml

```diff
@@ -1,10 +1,12 @@
+
 source_id: 65
 
 common_name: HESS J1418-609
 gamma_names: [HESS J1418-609]
 fermi_names: []
 other_names:
```

Comment:
In the HESS source catalog HESS J1418-609 corresponds to Rabbit and G313.3+0.1 and not to Kookaburra.  -> See comment in source 66 as well

Reply:
I think "Kookaburra" isn't really a well-defined thing. The best explanation I could find is in
https://www.aanda.org/articles/aa/full/2006/34/aa5511-06/aa5511-06.html
which says it's a complex region and both HESS J1418 and J1420 are in the Kookaburra region.
So I'm putting "Kookaburra" in `other_names` for both.